packages/grafana-ui/src/components/Logs/LogDetailsRow.tsx

```diff
@@ -69,43 +72,42 @@ class UnThemedLogDetailsRow extends PureComponent<Props, State> {
     });
   }
 
-  createStatsForLogs() {
-    const { getRows, parser, field } = this.props;
-    if (field && getRows) {
-      const allRows = getRows();
-
-      // Build value-agnostic row matcher based on the field label
-      const fieldLabel = parser!.getLabelFromField(field);
-      const fieldValue = parser!.getValueFromField(field);
+  createStatsForLabels() {
+    const { getRows, parser, parsedKey, parsedValue } = this.props;
+    // If is filterable, then isLabel = true
+    const isLabel = this.props.onClickFilterLabel;
```

Comment:
Can we make `isLabel` a prop so that the parent has to determine this? Then we can also use the prop to send it to the stats component to adapt the stats headline to be `M of N rows have that field` instead of `... have that label`

Reply:
👍absolutely! Great idea with the adaptive headline.